Mathlib/RingTheory/IntegralRestrict.lean

```diff
@@ -0,0 +1,121 @@
+/-
+Copyright (c) 2023 Andrew Yang, Patrick Lutz. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Andrew Yang
+-/
+import Mathlib.RingTheory.IntegrallyClosed
+import Mathlib.RingTheory.Norm
+import Mathlib.RingTheory.DedekindDomain.IntegralClosure
+/-!
+# Restriction of various maps between fields to integrally closed subrings.
+
+In this file, we assume `A` is an integrally closed domain; `K` is the fraction ring of `A`;
+`L` is a finite (separable) extension of `K`; `B` is the integral closure of `A` in `L`.
+We call this the AKLB setup.
+
+## Main definition
+- `galRestrict`: The restriction `Aut(L/K) → Aut(B/A)` as an `MulEquiv` in an AKLB setup.
```

Comment:
Can you add here and in the definition of `galRestrictHom` that `galRestrict` should be preferred over `galRestrictHom`?

Reply:
I think it is self evident since `galRestrictHom` is about `L →ₐ[K] L` where `galRestrict` is about `L ≃ₐ[K] L`. I think the reference of `galRestrict` in the docstring of `galRestrictHom` is enough?